refinery/core/api.py

```diff
@@ -678,24 +671,34 @@ def filter_by_group(self, request, obj_list):
         return obj_list
 
     def obj_get(self, bundle, **kwargs):
-        dataset = SharableResourceAPIInterface.obj_get(self, bundle, **kwargs)
-        return self.handle_incomplete_dataset(dataset)
+        try:
+            dataset = DataSet.objects.get(uuid=kwargs["uuid"])
```

Comment:
right, it actually should be:
```python
dataset = super(DataSetResource, self).obj_get(bundle, **kwargs)
```
with `SharableResourceAPIInterface` as the first parent in `DataSetResource` class definition to enable the correct method resolution order